tests/src/test/scala/whisk/core/database/test/CleanUpWhisksDbSkriptTests.scala

```diff
@@ -58,26 +58,35 @@ class CleanUpWhisksDbSkriptTests
         whisksDbName,
         "--dbNameSubjects",
         subjectsDbName,
+        "--whiskDBType",
+        whiskDbType,
         "--days",
-        "1",
+        "7",
         "--docsPerRequest",
         "1")
 
     val rr = TestUtils.runCmd(0, new File("."), cmd: _*)
 
-    val Seq(marked, deleted, skipped, kept) =
-      Seq("marking: ", "deleting: ", "skipping: ", "keeping: ").map { linePrefix =>
+    rr.stdout.lines.foreach {
+      case (line) =>
+        println(s"line: $line")
+    }
+
+    val Seq(marking, marked, deleting, skipping, keeping, statistic) =
+      Seq("marking: ", "marked: ", "deleting: ", "skipping: ", "keeping: ", "statistic: ").map { linePrefix =>
         rr.stdout.lines.collect {
           case line if line.startsWith(linePrefix) => line.replace(linePrefix, "")
         }.toList
       }
 
-    println(s"marked:  $marked")
-    println(s"deleted: $deleted")
-    println(s"skipped: $skipped")
-    println(s"kept:    $kept")
+    println(s"marked for deletion: $marking")
+    println(s"already marked:      $marked")
+    println(s"deleted:             $deleting")
```

Comment:
As we already print out every single line above, can we leave these printlns away, to not make these tests too noisy?

Reply:
Ok I will remove the code fragment to print out every single line (I used it only for debugging), so we are then back to the original behavior of the script.